fighters/common/src/general_statuses/turn.rs

```diff
@@ -86,6 +92,8 @@ unsafe extern "C" fn status_turn_main(fighter: &mut L2CFighterCommon) -> L2CValu
         if fighter.global_table[STICK_X].get_f32() == 0.0 {
             VarModule::off_flag(fighter.battle_object, vars::common::DISABLE_BACKDASH);
         }
+        // uncomment for pivot boosts
```

Comment:
for reference, is this the block which would add artificial speed within a small window (first few frames only) which could properly be called "perfect pivot"?